modules/src/ping.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+
+from templates.text import TextTemplate
+
+def process(input, entitites):
+    output = {}
+    try:
+        url = entities['url'][0]['value']
+        orig_url = url
+        strip_list = ['http://','https://','/']
```

Comment:
Add a space after comma. Can you explain why the third entry in the list is required?